File: PROG 1/ListaDeAtividade5/exemplos.py
import presets

def imprimir(vetor):
    i = 0
    while i < len(vetor):
        print("Nota %d = %.1f" % (i, vetor[i]))
        i = i + 1

def lerVetorNotas(vetNotas):
    stop = "N"
    while(stop == "N"):
        nota = presets.validNota()
        vetNotas.append(nota)
        stop = presets.stop()

# ----------------------FIM DAS FUNCTIONS---------------
def main():
    presets.head()

    meuVetor = [] 
    lerVetorNotas(meuVetor)
    imprimir(meuVetor)

    presets.copy()
main()

File: PROG 2/listaDeAtividade10/exemplos.c
#include <stdlib.h>
#include <stdio.h>
#include "functions.c"


int main(){
    head();
    int data[10] = {10, 20, 30,40, 50, 60, 70, 80, 90, 100};
    int *p;
    p = data;

    copy();
    return 0;
}
